inlang/guides/build-a-global-svelte-app/build-a-global-svelte-app.md

```diff
@@ -77,7 +77,7 @@ export default defineConfig({
 })
 ```
 
-With the Vite-Plugin added, you can also remove the `paraglide-js compile` from your package.json, if you added those. The plugin will take care of that.
```

Comment:
I found that the `paraglide-js compile` command is added automatically during `init`, so it's there even when the use doesn't add it